bin/reth/src/commands/node/mod.rs

```diff
@@ -217,7 +217,11 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
         let handle = node_config.launch::<Ext>(ext, executor).await?;
 
         // wait for node exit
-        handle.wait_for_node_exit().await
+        let node_exit_future = NodeExitFuture {
```

Comment:
@mattsse this seems worse / more verbose than before, why did we want this change?

Reply:
see comment above, this should replace the existing fields